### PR TITLE
Create build.yml for GitHub Actions

### LINKED ISSUE #68: [FEATURE]: Tests to run on CI (GitHub Actions)
### Description of the feature, you want to add.

Tests not running on push or pull request.

### Mention if this feature is for any specific Working Environment (e.g. operating system, browser, device, N/A)

NA

### Please add some pictorial references to the feature, if applicable (Drag & Drop your ss below)

NA

### What will be your aprroach to add this feature?

Added to GitHub Actions

### Any other context you have, to add here regarding this feature.

NA

### PR DESCRIPTION
**Pull request by ShaileshKumar007**
Added tests for GitHub Actions
closes #68

name: pull request
description: Create a pull request
title: "[PULL REQUEST] <pull request details>"
labels: [enhancement]
body:
  - type: textarea
    id: description
    attributes:
      label: Description of the changes, you have made.(Mention the issue number, it solves.)
    validations:
      required: true
  - type: textarea
    id: workingenvironment
    attributes:
      label: Mention if the changes are for any specific Working Environment (e.g. operating system, browser, device, N/A) 
    validations:
      required: true
  - type: textarea
    id: screenshots
    attributes:
      label: Please add some pictorial references to these changes, if applicable (Drag & Drop your ss below)
    validations:
      required: false
  - type: textarea
    id: solution
    attributes:
      label: Does your code changes make any other feature broke? (If Yes, State reason)
    validations:
      required: true
  - type: textarea
    id: context
    attributes:
      label: Any other context or any note for reviewer, to add here regarding this change.
    validations:
      required: false
  - type: markdown
    attributes:
      value: |
        You can also join our [Discord community](https://discord.gg/druweDMn3s)
        Feel free to check out other cool repositories of the Design and Code Community [here](https://github.com/Design-and-Code)

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: Build
+on: [push, pull_request]
+
```
Comment (thuongtruong1009) [inline .github/workflows/build.yml:7]: You should add branch name, where to want to build
Such as branch: [main, master]

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+      - name: install dependencies
```
Comment (thuongtruong1009) [inline .github/workflows/build.yml:16]: That also like npm install

**Review by thuongtruong1009 — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 2 above)
Comment (ShaileshKumar007) [reply .github/workflows/build.yml:16]: Should I change "npm ci" to "npm install" ?

**Comment by ShaileshKumar007** [comment]
I've added branch name in build.yml file. Let me know if any further changes required.

File: .github/workflows/build.yml (hunk 1 above)
Comment (ShaileshKumar007) [reply .github/workflows/build.yml:7]: Ok. I've added branch name. Let me know if any changes are required in this.

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        run: npm ci
+      - name: run tests
+        run: npm run test:e2e
```
Comment (sidwebworks) [inline .github/workflows/build.yml:33]: I am not sure if having a CI CD workflow without any test will make much sense at the moment.

Afterall, `test:e2e` script doesn't exist. So let's first get the initial features done and then add some test cases, then we can worry about CI CD workflows

**Pull request closed without merging**